test/production/app-dir/client-components-tree-shaking/app/client-reexport-index/page.js

```diff
@@ -0,0 +1,11 @@
+import ClientDefault from './client-module'
```

Comment:
Can we try importing `ClientModExportA` too just to see if that's kept as well?

Reply:
It would still be included as it's treated as cjs module, like the case in desc where we can't optimize